### DISCUSSION TITLE
What permissions does SEQ require from it's storage

### DISCUSSION THREAD

**Discussion by Talha-Altair** [datalust/seq-tickets#1849] [Help] [answered]
I have a PVC running on an on-prem k8s cluster, this particular PVC is backed by a NFS server (TrueNAS).

The server fails to start with this error, I suspect the error is with the storage, becoz SEQ seems to work fine with other NFS solutions like self-hosted or GCP filestore etc. But when the PVC Is backed by TrueNAS , the server fails with this error.

Any Idea what I am missing or what can be done to configure the storage to accept SEQ lock.

![image](https://github.com/datalust/seq-tickets/assets/70721398/2f6136e8-eb05-4d49-bdb9-4a819ae5d4a1)

**Comment by nblumhardt** [comment, marked as answer]
Hi @Talha-Altair, thanks for your message.

I don't think this will turn out to be a NAS configuration issue; some filesystems (I'd guess the one you are using may be among them) are not designed for hosting database software like Seq. Even if you're able to convince the filesystem to implement exclusive locking, there's a very good chance you'll hit data corruptions further down the road. This failure looks like an early warning, to me :-)

If you have any other filesystem option available, choosing it would be the way to go, here.

Hope this helps,
Nick

**Reply by Talha-Altair** [reply]
Thanks @nblumhardt
